### PR TITLE
CK3 Duchy Compatibility - Part B

### PR DESCRIPTION
**Pull request by IohannesIohannium**

### CODE REVIEW

File: EU4toV2/Data_Files/configurables/ck2_title_map.txt [hunk 1]
```diff
@@ -3664,7 +3664,7 @@ link = { name = "mazedonien" title = "d_mkd" region = "e_byzantium" }
 # Empires
 link = { name = "ajuraan" title = "e_ajuraan" region = "e_ajuraan" }
 link = { name = "ajuran" title = "e_ajuraan" region = "e_ajuraan" }
-link = { name = "guinea" title = "e_guinea" region = "e_guinea" }
+link = { name = "guinea" title = "e_guinea" region = "e_guinea }
```
Comment (Zemurin) [inline EU4toV2/Data_Files/configurables/ck2_title_map.txt:3667]: Close brace after Guinea

File: EU4toV2/Data_Files/configurables/country_mappings.txt [hunk 2]
```diff
@@ -457,6 +457,7 @@ link = { eu4 = LUX v2 = LUX } # Luxembourg
 link = { eu4 = d_luxembourg v2 = LUX } # Luxembourg
 link = { eu4 = NED v2 = NET } # Netherlands
 link = { eu4 = UTR v2 = UTR } # Utrecht
+link = { eu4 = d_utr v2 = UTR } # Utrecht
```
Comment (Zemurin) [inline EU4toV2/Data_Files/configurables/country_mappings.txt:460]: d_utr doesn't seem right

File: EU4toV2/Data_Files/configurables/country_mappings.txt [hunk 3]
```diff
@@ -1456,6 +1457,9 @@ link = { eu4 = d_tafilalt v2 = TFL } # Tafilalt
 link = { eu4 = d_udmurtia v2 = UDM } # Udmurtia
 link = { eu4 = d_uzboy v2 = UZB } # Uzboy => Uzbekistan
 link = { eu4 = d_yatenga v2 = YAT } # Yatenga
+link = { eu4 = d_asy v2 = ASY } # Chaldea => Assyria
```
Comment (Zemurin) [inline EU4toV2/Data_Files/configurables/country_mappings.txt:1460]: Also these 3 are off

**Review by Zemurin — CHANGES_REQUESTED** [review]

**Comment by IohannesIohannium** [comment]
d_utr and pals are intentional and they work

Il Mar 6 Ott 2020, 21:48 Ivo Elezović <notifications@github.com> ha scritto:

**Comment by Zemurin** [comment]
No worry, I'll fix guinea then (again, your last commit had the same error), but please fix it locally or sync after I do.

**Review by Zemurin — APPROVED** [review]

**Pull request merged**